gamestonk_terminal/helper_funcs.py

```diff
@@ -810,3 +810,49 @@ def inner(*args, **kwargs):
 def system_clear():
     """Clear screen"""
     os.system("cls||clear")  # nosec
+
+
+def log_execution(func=None, log=None):
```

Comment:
Can we give it a more descriptive name?

Maybe log_start_end?

Reply:
> Can we give it a more descriptive name?
> 
> Maybe log_start_end?

`log_execution_time`?